app/models/comment.rb

```diff
@@ -20,7 +20,7 @@ class Comment < ApplicationRecord
   validates :commentable_type, inclusion: { in: %w[Article PodcastEpisode] }
   validates :user_id, presence: true
 
-  after_create   :after_create_checks
```

Comment:
This can remain `after_create` now since we dont have to worry about relooking up the comment.

Reply:
Done